### ISSUE TITLE
Log plugin - Filter is slow

### ISSUE THREAD

**Issue by davsclaus** [hawtio/hawtio#1166]
We need a way for the filter to react faster. Maybe it do a full filter for every key stroke, instead it should maybe only start filtering if 3+ chars (like some other filters does) and also allow a bit delay before starting so people can type the keyword they want to search, so the filter do not start before the keyword has been fully typed.

Currently this fells sluggish and the filter do not react.

The screenshot show the filter with only 1 char

![screen shot 2014-03-25 at 11 22 14 am](https://f.cloud.github.com/assets/477100/2510434/ed135afa-b406-11e3-92bc-8ec63c2050aa.png)

**Comment by tadayosi** [comment]
Let's raise a new issue if we conflrm it's also related to Hawtio v2.
